File: app/src/main/java/com/dhsdevelopments/potato/clientapi/notifications/PotatoNotificationResult.java
package com.dhsdevelopments.potato.clientapi.notifications;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class PotatoNotificationResult
{
    @SerializedName( "event" )
    public String eventId;

    @SerializedName( "data" )
    public List<PotatoNotification> notifications;
}

File: app/src/main/java/com/dhsdevelopments/potato/clientapi/notifications/PotatoNotificationResult.kt
package com.dhsdevelopments.potato.clientapi.notifications

import com.google.gson.annotations.SerializedName

class PotatoNotificationResult {
    @SerializedName("event")
    lateinit var eventId: String

    @SerializedName("data")
    var notifications: List<PotatoNotification>? = null
}
